hack/agent_installer/start_db.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -euo pipefail
+set -o errexit
+
+/usr/bin/pg_ctl -D /tmp/postgres/data/ -l /tmp/postgres/logfile start 
```

Comment:
To be on the safe side I suggest we pass the `-w` (wait) flag explicitly. Looks like that might have become the default at some point but I'm not sure when.

Reply:
Done. It is there by default with `pg_ctl`, but still added for clarity and to be safe.
`-w, --wait             wait until operation completes (default)`